src/host/os_uuid.c

```diff
@@ -7,7 +7,7 @@
 #include "premake.h"
 
 #if PLATFORM_WINDOWS
-#include <Objbase.h>
+#include <objbase.h>
```

Comment:
Was this causing build issues? If not, I'd consider it out of scope of this PR.

Reply:
Compiler error:

```
src/host/os_uuid.c:10:10: fatal error: Objbase.h: No such file or directory
   10 | #include <Objbase.h>
      |          ^~~~~~~~~~~
```